drivers/macvlan/macvlan_setup.go

```diff
@@ -160,6 +169,15 @@ func parseVlan(linkName string) (string, int, error) {
 	return parent, vidInt, nil
 }
 
+// dummyLinkExists checks if dummylink exists in the default namespace
+func dummyLinkExists(dummyName string) bool {
```

Comment:
same function exist in ipvlan setup code. can we move into single common place. its redundant code.

Reply:
same comment as above